model_utils/network.py

```diff
@@ -183,18 +198,32 @@ def bidirectional_simple_rnn_bn_layer(name, input, size, share_weights):
             moving_mean_name=name + '_reverse_batch_norm_moving_mean',
             moving_variance_name=name + '_reverse_batch_norm_moving_variance')
         # forward and backward in time
-        forward_rnn = simple_rnn(
-            input=input_proj_bn_forward,
-            size=size,
+        forward_cell = RNNCell(
+            hidden_size=size,
+            activation=fluid.layers.brelu,
             param_attr=fluid.ParamAttr(name=name + '_forward_rnn_weight'),
-            bias_attr=fluid.ParamAttr(name=name + '_forward_rnn_bias'),
-            is_reverse=False)
-        reverse_rnn = simple_rnn(
-            input=input_proj_bn_backward,
-            size=size,
+            bias_attr=fluid.ParamAttr(name=name + '_forward_rnn_bias'))
+
+        pad_value = fluid.layers.assign(input=np.array([0.0], dtype=np.float32))
+        input, length = fluid.layers.sequence_pad(input_proj_bn, pad_value)
```

Comment:
`input_proj_bn` is not defined in `else` branch, please remember to test all the branches before pull request. And also please try to reuse these lines with the `if` branch.

Reply:
Done